css/style.css

```diff
@@ -122,33 +120,49 @@ body {
 
    p {
     color: #121212;
-    font-size: 13px;
+    font-size: 20px;
     font-weight: light;
     width: 400px;
     padding-top: 50px;
     font-kerning: 1px;
     text-align: center;
    }
 
+   header {
+     background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
+     background-size: cover;
+     height: 400px;
+     padding: 25px;
+   }
+
+#benefits p, #benefits ul {
+  width: 600px;
+}
+#benefits ul {
+  list-style: disc;
+  margin: 30px 0 0 80px;
+}
+
   .article {      
+    width: 100%;
+    margin: 0 auto;
     background-color: #f0efef;    
-    height: 300px;
-    top: 600px;
-    margin-bottom: 20px;
+   }   
 
-   }
-
-   ul {
-    text-align: center;
-    width: 400px;
-   }
+   .article nav ul li {
+    display: inline-block;
+    margin: 0 0 0 40px;
+    font-size: 24px;      
```

Comment:
@nyvangord add a little padding on the left to get it to match the comp. For example `padding-left: 10px; ` 🔨